src/compiler/fold_scale_axis.cc

```diff
@@ -347,12 +347,18 @@ NNVM_REGISTER_OP(relu)
 NNVM_REGISTER_OP(leaky_relu)
 .set_attr<FScaleAxisBackward>("FScaleAxisBackward", ReluScaleAxisBackward);
 
+NNVM_REGISTER_OP(prelu)
+.set_attr<FScaleAxisBackward>("FScaleAxisBackward", ReluScaleAxisBackward);
```

Comment:
PReLU don't have the scale axis property, because the channel wise weight. please remove it for now from fold_axis optimization